### PR TITLE
PPS Article refactored

### PR DESCRIPTION
**Pull request by vv-monsalve**
Hi @chrissimpkins, following what we discussed today, as well as taking one [previous font including an Article as a referent](https://fonts.google.com/specimen/Tiro+Kannada/about), I've rearranged the entire About section to give a better structure doing the following:

- [x] Add the "standard" (less editorial) description at the top of the article.
- [x] Including an `<hr>` element to divide the before from the Article text
- [x] Attempting to add a heading for the Article
- [x] Rearrange the text to move the mention of the Emojis under a subsection dedicated to them
- [x] Adding the list of the available emojis.
- [x] Final round on spell check grammar and turn it into third person.

### CODE REVIEW

**Comment by vv-monsalve** [comment]
@RosaWagner I think #7006 (the PR that is updating the font) should be merged first so this Article update overrides the Article included there.

File: ofl/playpensans/article/e-birthday-cake.png
Comment (chrissimpkins) [inline ofl/playpensans/article/e-birthday-cake.png:1]: This png is no longer required & should be removed

File: ofl/playpensans/article/e-color-palette.png
Comment (chrissimpkins) [inline ofl/playpensans/article/e-color-palette.png:1]: This png is no longer required & should be removed

File: ofl/playpensans/article/e-index-pointing.png
Comment (chrissimpkins) [inline ofl/playpensans/article/e-index-pointing.png:1]: This png is no longer required & should be removed

File: ofl/playpensans/article/e-shooting-star.png
Comment (chrissimpkins) [inline ofl/playpensans/article/e-shooting-star.png:1]: This png is no longer required & should be removed

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

File: ofl/playpensans/article/e-shooting-star.png
Comment (RosaWagner) [reply ofl/playpensans/article/e-shooting-star.png:1]: the png are already removed (red minus next to the file path), therefore considering this resolved.

**Comment by RosaWagner** [comment]
There will be some conflict so I gonna do some little tweaks to the first PR so I can merge this one after.

**Review by RosaWagner — APPROVED** [review]

**Comment by vv-monsalve** [comment]
I believe this was already stated in my previous comment.

**Comment by eliheuer** [comment]
Thanks, great example of why reading through everything PR to see what is linked is error prone.

**Pull request merged**